ci/nightly/pipeline.template.yml

```diff
@@ -1608,21 +1608,22 @@ steps:
         agents:
           queue: linux-aarch64-small
 
-  - group: ":bazel: Bazel"
-    key: bazel
-    steps:
-      - id: bazel-test-x86_64
-        label: ":bazel: Test (x86_64)"
-        command: bin/ci-builder run stable bazel test //... --remote_cache=https://bazel-remote.dev.materialize.com
-        depends_on: []
-        timeout_in_minutes: 60
-        agents:
-          queue: builder-linux-x86_64
-
-      - id: bazel-test-aarch64
-        label: ":bazel: Test (aarch64)"
-        command: bin/ci-builder run stable bazel test //... --remote_cache=https://bazel-remote.dev.materialize.com
-        depends_on: []
-        timeout_in_minutes: 60
-        agents:
-          queue: builder-linux-aarch64
+  # TODO(parkmycar): Re-enable when we auto-generate BUILD.bazel files for Cargo.toml.
```

Comment:
Do we happen to have an issue for this? If so, can we reference it here?

Reply:
Just made one and updated the PR!